bin/cli.js

```diff
@@ -42,6 +48,13 @@ function rollbackOpts (yargs) {
     .choices('env', ['dev', 'qa', 'prod'])
 }
 
+function applyOpts (yargs) {
+  return yargs
+    .describe('file', 'relative path of a file to be run on a cluster')
```

Comment:
can you also do alias('file', f) please?

Reply:
oh, that's neat, just added